Packs/ThreatConnect/ReleaseNotes/3_0_4.md

```diff
@@ -0,0 +1,8 @@
+
+#### Integrations
+
+##### ThreatConnect v3
+
+- Added new argument *owner_name* to the **tc-create-event** command.
+- Removed unnecessary *description* argument from the **tc-create-event** command since it is not supported in the API.
```

Comment:
```suggestion
- Added new argument *owner_name* to the ***tc-create-event*** command.
- Removed unnecessary *description* argument from the ***tc-create-event*** command since it is not supported in the API.
```